### ISSUE TITLE
IDLE UE STATE. No NAS PDU found ue error 

### ISSUE THREAD

**Issue by edaspb** [magma/magma#5861]
### Your Environment

- Version: 1.4.0
- Affected Component: Access Gateway
- Affected Subcomponent: MME
- Deployment Environment:*, bare-metal (AGW)
### Describe the Issue

I'm not sure about what exactly the issue but on both of my deployments I see a cycling logs when UE goes IDLE:

```
001139 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0720   [710700000000012] Sending NAS Establishment Indication to NAS for ue_id = (11)
001140 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_REQ (205)
001141 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0741    EMMAS-SAP - Received AS connection establish request
001142 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0882    EMMAS-SAP - Message Type = SERVICE_REQUEST(0x4d) for (ue_id = 11)
001143 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_recv.c    :0652    EMMAS-SAP - Received Service Request message for (ue_id = 11)
001144 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_CNF (206)
001145 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :2021    EMMAS-SAP - Send AS connection establish confirmation for (ue_id = 11)
001146 Wed Mar 31 15:26:53 2021 7F83A60E1700 ERROR MME-AP tasks/mme_app/mme_app_bearer.c  :0430   [710700000000012] No NAS PDU found ue 0x0000000B
001147 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1542   [710700000000012] UE STATE - CONNECTED.
001148 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0535   [710700000000012] MME APP : Sent Initial context Setup Request and Started guard timer for UE id  11 timer_id :601000001760 
001149 Wed Mar 31 15:26:53 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0865    Received Connection Establishment Confirm from MME_APP for ue_id = 11
001150 Wed Mar 31 15:26:53 2021 7F83A30DB700 NOTIC S1AP   tasks/s1ap/s1ap_mme_nas_procedur:1075   [710700000000012] Send S1AP_INITIAL_CONTEXT_SETUP_REQUEST message MME_UE_S1AP_ID = 0x0000000B eNB_UE_S1AP_ID = 0x02539F
001151 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1549    Received MME_APP_INITIAL_CONTEXT_SETUP_RSP from S1AP for ue_id = (11)
001152 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1479   [710700000000012] Sending S11 MODIFY BEARER REQ to SPGW for ue_id = (11), teid = (7)
001153 Wed Mar 31 15:26:53 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_main.c    :0121    Received S11 MODIFY BEARER RESPONSE from SPGW
001154 Wed Mar 31 15:26:59 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_handlers.c  :1068    UE CONTEXT RELEASE REQUEST with Cause_Type = Radio Network and Cause_Value = 20
001155 Wed Mar 31 15:26:59 2021 7F83A48DE700 INFO  GTPv1- lib/openflow/controller/PagingAp:0116    Added paging flow rule for UE IP 10.231.0.13
001156 Wed Mar 31 15:26:59 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_itti_messa:0089   [710700000000012] Sending UE Context Release Cmd to S1ap for (ue_id = 11)
UE Context Release Cause = (3)
001157 Wed Mar 31 15:26:59 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1956    Received UE context release complete message for ue_id: 0x0000000B
001158 Wed Mar 31 15:26:59 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1488   [710700000000012] UE STATE - IDLE.
001159 Wed Mar 31 15:27:28 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_handlers.c  :3135    Sent paging message over sctp for IMSI 710700000000012
001160 Wed Mar 31 15:27:29 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0102    Received S1AP INITIAL_UE_MESSAGE ENB_UE_S1AP_ID 0x02C54C assoc-id:2 
001161 Wed Mar 31 15:27:29 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0115    New Initial UE message received with eNB UE S1AP ID: 0x02C54C assoc-id :2 
001162 Wed Mar 31 15:27:29 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_itti_messagi:0175    Sending Initial UE Message to MME_APP, enb_ue_s1ap_id : 0x02C54C
001163 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0552    Received MME_APP_INITIAL_UE_MESSAGE from S1AP
001164 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0713   [710700000000012] INITIAL_UE_MESSAGE RCVD 
mme_ue_s1ap_id  = 11
enb_ue_s1ap_id  = 181580
001165 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0720   [710700000000012] Sending NAS Establishment Indication to NAS for ue_id = (11)
001166 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_REQ (205)
001167 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0741    EMMAS-SAP - Received AS connection establish request
001168 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0882    EMMAS-SAP - Message Type = SERVICE_REQUEST(0x4d) for (ue_id = 11)
001169 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_recv.c    :0652    EMMAS-SAP - Received Service Request message for (ue_id = 11)
001170 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_CNF (206)
001171 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :2021    EMMAS-SAP - Send AS connection establish confirmation for (ue_id = 11)
**001172 Wed Mar 31 15:27:29 2021 7F83A60E1700 ERROR MME-AP tasks/mme_app/mme_app_bearer.c  :0430   [710700000000012] No NAS PDU found ue 0x0000000B**
001173 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1542   [710700000000012] UE STATE - CONNECTED.
001174 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0535   [710700000000012] MME APP : Sent Initial context Setup Request and Started guard timer for UE id  11 timer_id :601000007010 
001175 Wed Mar 31 15:27:29 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0865    Received Connection Establishment Confirm from MME_APP for ue_id = 11
001176 Wed Mar 31 15:27:29 2021 7F83A30DB700 NOTIC S1AP   tasks/s1ap/s1ap_mme_nas_procedur:1075   [710700000000012] Send S1AP_INITIAL_CONTEXT_SETUP_REQUEST message MME_UE_S1AP_ID = 0x0000000B eNB_UE_S1AP_ID = 0x02C54C
001177 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1549    Received MME_APP_INITIAL_CONTEXT_SETUP_RSP from S1AP for ue_id = (11)
001178 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1479   [710700000000012] Sending S11 MODIFY BEARER REQ to SPGW for ue_id = (11), teid = (7)
001179 Wed Mar 31 15:27:29 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_main.c    :0121    Received S11 MODIFY BEARER RESPONSE from SPGW
001180 Wed Mar 31 15:27:35 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_handlers.c  :1068    UE CONTEXT RELEASE REQUEST with Cause_Type = Radio Network and Cause_Value = 20
001181 Wed Mar 31 15:27:35 2021 7F83A48DE700 INFO  GTPv1- lib/openflow/controller/PagingAp:0116    Added paging flow rule for UE IP 10.231.0.13
001182 Wed Mar 31 15:27:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_itti_messa:0089   [710700000000012] Sending UE Context Release Cmd to S1ap for (ue_id = 11)
UE Context Release Cause = (3)
001183 Wed Mar 31 15:27:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1956    Received UE context release complete message for ue_id: 0x0000000B
001184 Wed Mar 31 15:27:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1488   [710700000000012] UE STATE - IDLE.
001185 Wed Mar 31 15:27:59 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_handlers.c  :3135    Sent paging message over sctp for IMSI 710700000000012
001186 Wed Mar 31 15:28:00 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0102    Received S1AP INITIAL_UE_MESSAGE ENB_UE_S1AP_ID 0x036656 assoc-id:2 
001187 Wed Mar 31 15:28:00 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0115    New Initial UE message received with eNB UE S1AP ID: 0x036656 assoc-id :2 
001188 Wed Mar 31 15:28:00 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_itti_messagi:0175    Sending Initial UE Message to MME_APP, enb_ue_s1ap_id : 0x036656
001189 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0552    Received MME_APP_INITIAL_UE_MESSAGE from S1AP
001190 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0713   [710700000000012] INITIAL_UE_MESSAGE RCVD 
mme_ue_s1ap_id  = 11
enb_ue_s1ap_id  = 222806
001191 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0720   [710700000000012] Sending NAS Establishment Indication to NAS for ue_id = (11)
001192 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_REQ (205)
001193 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0741    EMMAS-SAP - Received AS connection establish request
001194 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0882    EMMAS-SAP - Message Type = SERVICE_REQUEST(0x4d) for (ue_id = 11)
001195 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_recv.c    :0652    EMMAS-SAP - Received Service Request message for (ue_id = 11)
001196 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_CNF (206)
001197 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :2021    EMMAS-SAP - Send AS connection establish confirmation for (ue_id = 11)
**001198 Wed Mar 31 15:28:00 2021 7F83A60E1700 ERROR MME-AP tasks/mme_app/mme_app_bearer.c  :0430   [710700000000012] No NAS PDU found ue 0x0000000B**
001199 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1542   [710700000000012] UE STATE - CONNECTED.
001200 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0535   [710700000000012] MME APP : Sent Initial context Setup Request and Started guard timer for UE id  11 timer_id :601000004e60 
001201 Wed Mar 31 15:28:00 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0865    Received Connection Establishment Confirm from MME_APP for ue_id = 11
001202 Wed Mar 31 15:28:00 2021 7F83A30DB700 NOTIC S1AP   tasks/s1ap/s1ap_mme_nas_procedur:1075   [710700000000012] Send S1AP_INITIAL_CONTEXT_SETUP_REQUEST message MME_UE_S1AP_ID = 0x0000000B eNB_UE_S1AP_ID = 0x036656
001203 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1549    Received MME_APP_INITIAL_CONTEXT_SETUP_RSP from S1AP for ue_id = (11)
001204 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1479   [710700000000012] Sending S11 MODIFY BEARER REQ to SPGW for ue_id = (11), teid = (7)
001205 Wed Mar 31 15:28:00 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_main.c    :0121    Received S11 MODIFY BEARER RESPONSE from SPGW
001206 Wed Mar 31 15:28:10 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_handlers.c  :1068    UE CONTEXT RELEASE REQUEST with Cause_Type = Radio Network and Cause_Value = 20
001207 Wed Mar 31 15:28:10 2021 7F83A48DE700 INFO  GTPv1- lib/openflow/controller/PagingAp:0116    Added paging flow rule for UE IP 10.231.0.13
001208 Wed Mar 31 15:28:10 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_itti_messa:0089   [710700000000012] Sending UE Context Release Cmd to S1ap for (ue_id = 11)
UE Context Release Cause = (3)
001209 Wed Mar 31 15:28:10 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1956    Received UE context release complete message for ue_id: 0x0000000B
001210 Wed Mar 31 15:28:10 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1488   [710700000000012] UE STATE - IDLE.
001211 Wed Mar 31 15:28:35 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0102    Received S1AP INITIAL_UE_MESSAGE ENB_UE_S1AP_ID 0x04E140 assoc-id:2 
001212 Wed Mar 31 15:28:35 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0115    New Initial UE message received with eNB UE S1AP ID: 0x04E140 assoc-id :2 
001213 Wed Mar 31 15:28:35 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_itti_messagi:0175    Sending Initial UE Message to MME_APP, enb_ue_s1ap_id : 0x04E140
001214 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0552    Received MME_APP_INITIAL_UE_MESSAGE from S1AP
001215 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0713   [710700000000012] INITIAL_UE_MESSAGE RCVD 
mme_ue_s1ap_id  = 11
enb_ue_s1ap_id  = 319808
001216 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0720   [710700000000012] Sending NAS Establishment Indication to NAS for ue_id = (11)
001217 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_REQ (205)
001218 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0741    EMMAS-SAP - Received AS connection establish request
001219 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0882    EMMAS-SAP - Message Type = SERVICE_REQUEST(0x4d) for (ue_id = 11)
001220 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_recv.c    :0652    EMMAS-SAP - Received Service Request message for (ue_id = 11)
001221 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0176    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_CNF (206)
001222 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :2021    EMMAS-SAP - Send AS connection establish confirmation for (ue_id = 11)
**001223 Wed Mar 31 15:28:35 2021 7F83A60E1700 ERROR MME-AP tasks/mme_app/mme_app_bearer.c  :0430   [710700000000012] No NAS PDU found ue 0x0000000B**
001224 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1542   [710700000000012] UE STATE - CONNECTED.
001225 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :0535   [710700000000012] MME APP : Sent Initial context Setup Request and Started guard timer for UE id  11 timer_id :601000006ad0 
001226 Wed Mar 31 15:28:35 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0865    Received Connection Establishment Confirm from MME_APP for ue_id = 11
001227 Wed Mar 31 15:28:35 2021 7F83A30DB700 NOTIC S1AP   tasks/s1ap/s1ap_mme_nas_procedur:1075   [710700000000012] Send S1AP_INITIAL_CONTEXT_SETUP_REQUEST message MME_UE_S1AP_ID = 0x0000000B eNB_UE_S1AP_ID = 0x04E140
001228 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1549    Received MME_APP_INITIAL_CONTEXT_SETUP_RSP from S1AP for ue_id = (11)
001229 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_bearer.c  :1479   [710700000000012] Sending S11 MODIFY BEARER REQ to SPGW for ue_id = (11), teid = (7)
001230 Wed Mar 31 15:28:35 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_main.c    :0121    Received S11 MODIFY BEARER RESPONSE from SPGW
001231 Wed Mar 31 15:28:46 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_handlers.c  :1068    UE CONTEXT RELEASE REQUEST with Cause_Type = Radio Network and Cause_Value = 20
001232 Wed Mar 31 15:28:46 2021 7F83A48DE700 INFO  GTPv1- lib/openflow/controller/PagingAp:0116    Added paging flow rule for UE IP 10.231.0.13
001233 Wed Mar 31 15:28:46 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_itti_messa:0089   [710700000000012] Sending UE Context Release Cmd to S1ap for (ue_id = 11)
UE Context Release Cause = (3)
001234 Wed Mar 31 15:28:46 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1956    Received UE context release complete message for ue_id: 0x0000000B
001235 Wed Mar 31 15:28:46 2021 7F83A60E1700 INFO  MME-AP tasks/mme_app/mme_app_context.c :1488   [710700000000012] UE STATE - IDLE.
001236 Wed Mar 31 15:29:08 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_handlers.c  :3135    Sent paging message over sctp for IMSI 710700000000012
001237 Wed Mar 31 15:29:09 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0102    Received S1AP INITIAL_UE_MESSAGE ENB_UE_S1AP_ID 0x00CF45 assoc-id:2 
001238 Wed Mar 31 15:29:09 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_nas_procedur:0115    New Initial UE message received with eNB UE S1AP ID: 0x00CF45 assoc-id :2 
001239 Wed Mar 31 15:29:09 2021 7F83A30DB700 INFO  S1AP   tasks/s1ap/s1ap_mme_itti_messagi:0175    Sending Initial UE Message to MME_APP, enb_ue_s1ap_id : 0x00CF45
```
It can be repeated many times. The main service keeps working well.
Is it a bug or expected behavior? I have a concern about `No NAS PDU found ue` error.

**Comment by sudhikan** [comment]
@edaspb is there a pcap that goes along with this?

**Comment by ulaskozat** [comment]
@edaspb looks like RAN side is sending UE CONTEXT RELEASE REQUEST with cause value 20 (which indicates user inactivity). `No NAS PDU found` should be benign. As @sudhikan asked if you can send pcap, it would help a lot. Particularly, I want to see if there is any traffic sent by the UE after session is established. Ideally flow rules on table 0 and table 12 for the UE would be also good to have.

**Comment by edaspb** [comment]
Hi, thanks for your help. @ulaskozat, could you please guide me how to make a dump of flows for specific tables?
[tables.txt](https://github.com/magma/magma/files/6250192/tables.txt)
[flows.txt](https://github.com/magma/magma/files/6250194/flows.txt)
[sctp1.pcap.zip](https://github.com/magma/magma/files/6250197/sctp1.pcap.zip)

**Comment by ulaskozat** [comment]
@edaspb here is an example for table 12:
`sudo ovs-ofctl dump-flows gtp_br0 table=12`

**Comment by ulaskozat** [comment]
@edaspb, pcap is showing the signaling in one direction only (Magma to eNB). Do you have the full packet capture with both directions?

**Comment by edaspb** [comment]
At the moment of taking dumps there were two subscribers on AGW:
```
SID		IP		APN
IMSI710700000000011	10.231.1.114	internet
IMSI710700000000012	10.231.1.18	internet
```
The `No NAS PDU found` error message presents only for  IMSI710700000000011
[table12.txt](https://github.com/magma/magma/files/6250943/table12.txt)
[table0.txt](https://github.com/magma/magma/files/6250944/table0.txt)

**Comment by ulaskozat** [comment]
the table entries look correct to me. Table 0 has the paging rules, Table 12 has the gating rules and Table 13 has the stat rules for UE IPs 10.231.1.114 and 10.231.1.18.

On pcap, if we do not see any gtp traffic from eNB to AGW, then the behavior looks normal (i.e., eNB triggering context release due to UE inactivity and Magma releasing it).

**Comment by ulaskozat** [comment]
though what is peculiar is that for 10.231.1.18 the uplink counters are positive (table 12) but downlink counters are zero. Can you tell more about the set up, how many UEs are being used (looks like 2 only?), and whether the connectivity issue is transient or not, does the connectivity issue occur for a particular UE or for all, etc.

**Comment by edaspb** [comment]
1. We use multi apn scheme, 3 apns are configured, only one of them us in use.
2. DHCP lease time is 12 hours.
3. Max simultaneous UE on this GW was about 5, not many. At the moment of taking statistic there were two devices connected, you were right.
4. The issue is not transient unfortunately, I could reproduce it one time in office lab and never seen again. Looks like issue occur for particular UEs, we had two cases reported (from maybe ten test devices).

I found another weird thing in the log:
```
Apr  2 13:25:08 magma2 mobilityd[25927]: INFO:root:GW update: GW IP[1985]: 10.231.1.1 : mac 00:25:84:7c:69:40
Apr  2 13:25:09 magma2 mobilityd[25927]: INFO:root:GW update: GW IP[1984]: 10.231.0.1 : mac 00:25:84:7c:69:40
```
But IP 10.231.0.1 doesn't belong to that AGW neither to any of its gateways.
Apn configured with static IP-mac:

```
    "internet": {
      "apn_name": "internet",
      "gateway_ip": "10.231.1.1",
      "gateway_mac": "00:25:84:7c:69:40",
      "id": "internet_1985",
      "vlan_id": 1985
```

**Comment by ulaskozat** [comment]
@pshelar do you have any insights about this? seems like a DL connectivity issue on SGi. @edaspb it would greatly help to collect pcap on both eth0 (SGi) and eth1 (S1) interfaces. I want to see if for problematic UEs, their UL traffic goes onto eth0 and whether any traffic is coming back to SGi from the remote end point.

**Comment by edaspb** [comment]
@ulaskozat Ok, I will check that. Thanks a lot, now I know where to look at. Next week I'll try to find problematic UE, reproduce the issue and revise ip network part.

**Comment by edaspb** [comment]
I found one weird behavior which can be potential reason to the issue.
In a parallel we have another AGW (hostname gw2-1) which doesn't have active S1 session. We used it for S1-flex testing. Config is absolutely similar to the active gateway (hostname gw2-2) . Few days ago gw2-1 was active too and had few connected UE. It has been with no active subscribers at least for the last 48 hours.
But is still sends DHCP requests. Here is screenshot:
![image](https://user-images.githubusercontent.com/46284043/113468399-287c9c80-9403-11eb-99b2-870eefaf02af.png)

My notes:

1,2 No subscribers, no allocated IP
3. Active dhcp records, take a look to renew time
4. Date for reference.
5,6 Dhcp record after renew time from point 3. Timers are updated. On dhcp server request can be seen as well.
7. Still no active subscribers.

This dhcp requests brake our ip path logic and may cause the issue I described above.
